nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -0,0 +1,48 @@
+collision_monitor:
+  ros__parameters:
+    use_sim_time: True
+    base_frame_id: "base_footprint"
+    odom_frame_id: "odom"
+    cmd_vel_in_topic: "cmd_vel_raw"
+    cmd_vel_out_topic: "cmd_vel"
+    transform_tolerance: 0.5
+    data_timeout: 5.0
+    stop_pub_timeout: 2.0
+    # Polygons represent safety area around the robot for "stop" and "slowdown" action types,
+    # and robot footprint for "circle" action type.
+    # Footprint could be "polygon" type with dynamically set footprint from footprint_topic
+    # or "circle" type with static footprint set by radius. "footprint_topic" parameter
+    # to be ignored in latest case.
+    polygons: ["PolygonStop"]
+    PolygonStop:
+      type: "polygon"
+      points: [0.5, 0.5, 0.5, -0.5, -0.1, -0.5, -0.1, 0.5, 0.5, 0.5]
+      action_type: "stop"
+      max_points: 3
+      visualize: True
+      polygon_topic: "polygon_stop"
```

Comment:
How about `polygon_pub_topic`? I only ask because when I read this, I thought maybe this was like the `footprint_topic` where that's an input and was confused why we have a topic input for polygons but also the `points`.

Adding the `pub` makes it clear that this topic is only for publication of the footprint for visualization.